index.md

```diff
@@ -19,7 +19,7 @@ permalink: /
 
 **BurnP3+** was developed to improve scalability to larger landscapes and number of scenarios, allow for cross-compatibility among platforms (Windows and Linux) and interfaces (SyncroSim Studio, command line, R and Python), and increase flexibility in model structure through the implementation of modules. With these enhancements, **BurnP3+** extends the success of Burn-P3 as a decision-support tool in land management and a framework for scientific inquiry ([Parisien *et al.* 2019](https://www.fs.usda.gov/research/treesearch/60727){:target="_blank"}).
 
-Burn-P3 utilized a single deterministic fire growth model called [Prometheus](https://firegrowthmodel.ca/pages/prometheus_overview_e.html){:target="_blank"}. **BurnP3+** now provides two fire growth model : [BurnP3+Cell2Fire](https://github.com/BurnP3/BurnP3PlusCell2Fire){:target="_blank"} enables users to grow fires with [Cell2Fire](https://doi.org/10.3389/ffgc.2021.692706){:target="_blank"}; and [BurnP3+Prometheus](https://github.com/BurnP3/BurnP3PlusPrometheus){:target="_blank"} enables users to grow fires with [Prometheus](https://firegrowthmodel.ca/pages/prometheus_overview_e.html){:target="_blank"}. The Canadian Forest Service plans to develop additional fire growth packages for **BurnP3+**. Alternatively, users also have the option to develop their own fire growth models as SycnroSim packages. <br>
+Burn-P3 utilized a single deterministic fire growth model called [Prometheus](https://firegrowthmodel.ca/pages/prometheus_overview_e.html){:target="_blank"}. **BurnP3+** now provides two fire growth models: [Cell2Fire](https://github.com/BurnP3/BurnP3PlusCell2Fire){:target="_blank"} enables users to grow fires with [Cell2Fire](https://doi.org/10.3389/ffgc.2021.692706){:target="_blank"}; and [BurnP3Prometheus](https://github.com/BurnP3/BurnP3PlusPrometheus){:target="_blank"} enables users to grow fires with [Prometheus](https://firegrowthmodel.ca/pages/prometheus_overview_e.html){:target="_blank"}. The Canadian Forest Service plans to develop additional fire growth packages for **BurnP3+**. Alternatively, users also have the option to develop their own fire growth models as SycnroSim packages. <br>
```

Comment:
Sorry, can you change the package names to "BurnP3+ Cell2Fire" and "BurnP3+ Prometheus"?

Reply:
Also typo "SycnroSim"